bundles/ranvier-commands/commands/look.js

```diff
@@ -74,8 +74,15 @@ module.exports = (srcPath) => {
       if (otherPlayer === player) {
         return;
       }
-
-      Broadcast.sayAt(player, '[Player] ' + otherPlayer.name);
+      let combatantsDisplay = '';
+      if (otherPlayer.isInCombat()) {
+        const combatantsList = [...otherPlayer.combatants.values()].map(combatant => combatant.name);
+        combatantsDisplay += `, <red>fighting: </red>`
+        combatantsDisplay += (combatantsList.length > 1 ?
+           `<bold>[ </bold>${combatantsList.join(", ")}<bold> ]</bold>` :
```

Comment:
Do you mean the square [ ] brackets? I was doing that for presentation's sake but it'll also look less cluttered without them.